### PR TITLE
add not-supported deviation for lldp parameters

### PR DESCRIPTION
**Pull request by spenugondaa**
LLDP has currently support only for get. Need to deviate config parameters  as not-supported. 
([SNC-3923](https://jira.force10networks.com/browse/SNC-3923))

Added not-supported tag to lldp/config and lldp/interfaces/interface/config/enabled

Tried all show lldp commands through CLI and swagger to make sure nothing is broken.
Swagger does not display lldp/config and lldp/interfaces/interface/config/enabled.

image

Tested few basic tests on both system and interfaces yaml through swagger.
configuring tacacs server,
get on few interfaces

### CODE REVIEW

File: models/yang/extensions/openconfig-lldp-ext.yang [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module openconfig-lldp-ext {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/lldp/extention";
```
Comment (GarrickHe) [inline models/yang/extensions/openconfig-lldp-ext.yang]: typo here: 'extention' should be 'extension'

**Review by GarrickHe — CHANGES_REQUESTED** [review]

File: models/yang/extensions/openconfig-lldp-ext.yang [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  prefix "oc-lldp-ext";
+
+  import openconfig-lldp { prefix oc-lldp; }
+  import openconfig-lldp-types { prefix oc-lldp-types; }
+  import openconfig-interfaces { prefix oc-if; }
+  import ietf-yang-types { prefix yang; }
+  import openconfig-extensions { prefix oc-ext; }
```
Comment (GarrickHe) [inline models/yang/extensions/openconfig-lldp-ext.yang]: I only see `import openconfig-lldp { prefix oc-lldp; }` used. Are the rest of the imports needed?

Comment (spenugondaa) [reply models/yang/extensions/openconfig-lldp-ext.yang]: Removed the other imports.

File: models/yang/extensions/openconfig-lldp-ext.yang (hunk 1 above)
Comment (spenugondaa) [reply models/yang/extensions/openconfig-lldp-ext.yang]: Changed it to extension. Based it on interfaces ext yang.

**Review by GarrickHe — APPROVED** [review]
lgtm

**Comment by sameerdell** [comment]
WIll create a new PR.

**Pull request closed without merging**